tests/test_secret_scrubbing.py

```
@@ -14,9 +14,113 @@
+    Compare against that pattern in isolation, not the combined leftmost match,
+    so a new pattern cannot be silently disabled if its example also contains
+    an earlier default match, e.g. adding `token` with `has_secret_token_value`.
+    """
+    assert DEFAULT_PATTERN_EXAMPLES.keys() == set(DEFAULT_PATTERNS)
+    isolated_examples = [
+        *DEFAULT_PATTERN_EXAMPLES.items(),
+        (r'(?:\b|_)csrf(?:\b|_)', 'has_csrf_value'),
+        (r'(?:\b|_)xsrf(?:\b|_)', 'has_xsrf_value'),
+        (r'(?:\b|_)jwt(?:\b|_)', 'has_jwt_value'),
+        (r'(?:\b|_)ssn(?:\b|_)', 'has_ssn_value'),
+        ('secret', 'has ſecret value'),
```

Ruff (0.16.1)

[warning] String contains ambiguous `ſ` (LATIN SMALL LETTER LONG S). Did you mean `f` (LATIN SMALL LETTER F)?

(RUF001)